packages/shell-api/src/field-level-encryption.spec.ts

```diff
@@ -311,20 +307,20 @@ describe('Field Level Encryption', () => {
   });
   describe('Mongo constructor FLE options', () => {
     before(() => {
-      libmongoc = stubInterface<lmc>();
+      libmongoc = stubInterface<FLEClientEncryption>();
       sp = stubInterface<ServiceProvider>();
       sp.bsonLibrary = bson;
-      sp.fle = { ClientEncryption: function() { return libmongoc; } };
+      sp.fle = { ClientEncryption: function() { return libmongoc; } } as any;
       sp.initialDb = 'test';
       internalState = new ShellInternalState(sp, stubInterface<EventEmitter>());
       internalState.currentDb = stubInterface<Database>();
     });
     it('accepts the same local key twice', () => {
-      const localKmsOptions = {
+      const localKmsOptions: ClientSideFieldLevelEncryptionOptions = {
         keyVaultNamespace: `${DB}.${COLL}`,
         kmsProvider: {
           local: {
-            key: new bson.Binary(Buffer.alloc(96).toString('base64'))
+            key: Buffer.alloc(96).toString('base64')
```

Comment:
Ah, maybe there should have been a comment here leaving some context – this is a regression test case that makes sure that we don’t modify `kmsProvider.local.key` by converting it from `Binary` to `Buffer` in-place, so moving this to a plain `Buffer` more or less invalidates the test :)

Reply:
Ah okay :D I've put these tests into `shell-api`: https://github.com/mongodb-js/mongosh/pull/553/files/fce05fba6c22c207cd9aa6acf22dcb633b276ad1#diff-6567f9efa0de147b837a1b2c94bc14ca6b1f8d88dd7b5ccde016fa71735281bbR215 to make sure we support all 3 variants - binary, buffer, or plain string :) I think we can leave this as is then? 🤔

Reply:
Yeah, I saw that – so this should remain:
```suggestion
            key: new bson.Binary(Buffer.alloc(96).toString('base64'))
```

Reply:
It's back ✅